cms/blocks.py

```diff
@@ -11,3 +12,35 @@ class ResourceBlock(blocks.StructBlock):
 
     heading = blocks.CharBlock(max_length=100)
     detail = blocks.RichTextBlock()
+
+
+class FacultyBlock(blocks.StructBlock):
+    """
+    Block class that defines a faculty member
+    """
+
+    name = blocks.CharBlock(max_length=100, help_text="Name of the faculty member.")
+    image = ImageChooserBlock(
+        help_text="Profile image size must be at least 300x300 pixels."
+    )
+    description = blocks.RichTextBlock(
+        help_text="A brief description about the faculty member."
+    )
+
+
+class FacultMemberBlock(blocks.StructBlock):
```

Comment:
1. Typo
2. This should be renamed to `FacultySectionBlock`

Reply:
`Instructor` instead of `Faculty`